Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+    skipDefaultCheckout()  // see 'Checkout SCM' below, once perms are fixed this is no longer needed
+  } 
+
+  stages {
+    stage('Checkout SCM') {
+      steps {
+        sh 'sudo chown -R jenkins:jenkins .'  // bad docker mount creates unreadable files TODO fix this
```

Comment:
I didn't even wait to see if there was a problem here. I assumed that because I've got a container writing test results, I needed to do this.

I'll experiment with taking it out and seeing what happens.